src/Microsoft.ML.OnnxTransformer/OnnxUtils.cs

```diff
@@ -143,14 +143,9 @@ public OnnxVariableInfo(string name, OnnxShape shape, Type typeInOnnxRuntime, Da
         /// </summary>
         private readonly InferenceSession _session;
         /// <summary>
-        /// Indicates if <see cref="ModelFile"/> is a temporal file created by <see cref="CreateFromBytes(byte[], int?, bool, IDictionary{string, int[]})"/>
-        /// or <see cref="CreateFromBytes(byte[])"/>. If <see langword="true"/>, <see cref="Dispose(bool)"/> should delete <see cref="ModelFile"/>.
-        /// </summary>
-        private bool _ownModelFile;
-        /// <summary>
         /// The location where the used ONNX model loaded from.
```

Comment:
This comment is now out of date.

Reply:
Comment has been removed.

Reply:
The comment still isn't removed.

Reply:
Ok, now it says `The FileStream holding onto the loaded ONNX model.`. It will go with my next push.